modules/poll/poll.class.php

```diff
@@ -80,6 +114,16 @@ function moduleUpdate()
 	function recompileCache()
 	{
 	}
+
+	function isCangetMemberInfo($type)
```

Comment:
함수를 한 번 만들어 놓으면 이름을 바꾸기 어려우니, 외국인들이 조금만 더 자연스럽게 이해할 수 있도록 다듬어 봅시다. 메소드 이름은 `$this`를 주어로 놓았을 때 영어 문법에 맞도록 하는 것이 좋습니다. This is can get member info는 동사가 연속으로 3개나 나와서 이상하잖아요. This has member info 또는 This can have member info 정도로 하면 되겠습니다.

아래의 `isCanAddItem`도 마찬가지... This can add item?